pkg/exchange/binance/depthframe.go

```diff
@@ -146,6 +149,91 @@ func (f *DepthFrame) loadDepthSnapshot() error {
 	return nil
 }
 
+func (f *DepthFrame) loadFuturesDepthSnapshot() error {
```

Comment:
can we share the same depth frame function? looks like you're just calling a different client but it returns the same structure?

Reply:
Use interface to handle fetches from different clients ?